kolibri/plugins/learn/viewsets.py

```diff
@@ -182,15 +183,30 @@ def consolidate(self, items, queryset):
                 "title",
                 "closed",
                 "answer_count",
+                "data_model_version",
                 "score",
                 "question_sources",
             )
         )
         exam_node_ids = set()
+        section_questions = list()
+
         for exam in exams:
-            exam_node_ids |= {
```

Comment:
I mentioned this on slack, but this diff would be simpler:
```
from kolibri.core.exam.models import exam_assignment_lookup
```
then here:

```
            exam_node_ids |= {
                exercise_id for exercise_id, _ in exam_assignment_lookup(exam.get("question_sources", []))
            }
```

Reply:
Aye I hadn't considered that `exam_assignment_lookup` handles the backward compatibility for us , thanks!